docs/data/data-grid/scrolling/ScrollPlayground.tsx

```diff
@@ -27,7 +27,7 @@ export default function ScrollPlayground() {
     rowLength: 100,
   });
 
-  React.useEffect(() => {
+  React.useLayoutEffect(() => {
```

Comment:
Right, otherwise there's a warning on server about using `useLayoutEffect`.
If I understand correctly that's the only issue with `useLayoutEffect` on server, since both `useEffect` and `useLayoutEffect` are not executed on server.

I'll try to set up an Eslint rule for this later.